<!DOCTYPE html>

<html lang="en">


<head>
    <meta charset="UTF-8">
    <meta name="description" content="Home of Bioinformatics and Interdisciplinary Technologies (BITs)">
    <meta name="author" content="Bioinformatics and Interdisciplinary Technologies (BITs)">
    <link rel="shortcut icon" href="/static/image/favicon.ico">
    <title>Bioinformatics and Interdisciplinary Technologies (BITs) &mdash; Yang Lab</title>

    

    <link rel="stylesheet" href="/static/css/uikit.almost-flat.min.css" type="text/css">
    <link rel="stylesheet" href="/static/css/components/sticky.almost-flat.min.css" type="text/css">
    <script src="/static/js/jquery.min.js" type="text/javascript" charset="utf-8"></script>
    <script src="/static/js/uikit.min.js" type="text/javascript" charset="utf-8"></script>
    <script src="/static/js/components/sticky.min.js" type="text/javascript" charset="utf-8"></script>
    <script src="/static/js/components/grid.min.js" type="text/javascript" charset="utf-8"></script>
    
<link rel="stylesheet" href="/static/css/components/slideshow.almost-flat.min.css" type="text/css">
<link rel="stylesheet" href="/static/css/components/slidenav.almost-flat.min.css" type="text/css">
<script src="/static/js/components/slideshow.min.js" type="text/javascript" charset="utf-8"></script>


</head>

<body>

    

    <div class="uk-container uk-container-center uk-margin-large-top uk-margin-large-bottom">

        <div class="uk-grid" data-uk-grid-margin>
    <div class="uk-width-1-1">
        <!--<h1><img src="/static/image/favicon.png" alt="logo">Computational Biology and Biotechnologies (Yang Lab) <br> <text>on genomics, transcriptomics and genome editing<text></h1>-->
        <span>
        <div style="display:inline-block"><img src="/static/image/favicon.png" alt="logo"></div>
        <div style="display:inline-block">  
            <span style="font-size: 36px">Bioinformatics and Interdisciplinary Technologies (BITs), Yang Lab</span>
            <br>
        </div>
        </span>
    </div>
</div>

        <nav class="uk-navbar uk-margin-top uk-margin-bottom" data-uk-sticky="{top:-50}">
    <a class="uk-navbar-brand uk-hidden-small" href="#">Yang Lab</a>
    <ul class="uk-navbar-nav uk-hidden-small">
        <li><a href="/index"><i class="uk-icon-home"></i> Home</a></li>
       <!-- <li><a href="/research"><i class="uk-icon-laptop"></i> Research</a></li>  -->
        <li><a href="/news"><i class="uk-icon-newspaper-o"></i> News</a></li>
        
        <li><a href="/team"><i class="uk-icon-users"></i> Team</a></li>
        <li><a href="/publications"><i class="uk-icon-file-text"></i> Publications</a></li>

        <li><a href="/awards"><i class="uk-icon-star"></i> Awards</a></li>
        <!--<li><a href="/pictures"><i class="uk-icon-picture-o"></i> Pictures</a></li>-->

        <li><a href="/resources"><i class="uk-icon-wrench"></i> Resources</a></li>
        <!--<li><a href="/SM"><i class="uk-icon-cubes"></i> Supplementary Materials</a></li>-->
    </ul>
    <a href="#nav-offcanvas" class="uk-navbar-toggle uk-visible-small" data-uk-offcanvas></a>
    <div class="uk-navbar-brand uk-navbar-center uk-visible-small">Yang Lab</div>
</nav>

        <article class="uk-article">

            

<div>
    <div style="float:left;width:58.7%;" >
        <div class="uk-slidenav-position uk-margin-top uk-margin-bottom" data-uk-slideshow="{kenburns:'3s', autoplay:true, autoplayInterval: 3000}">
            <ul class="uk-slideshow">
                
                <li><img src="/static/image/pictures/program.jpg" alt="picture" width="800" height="400"></li>
                
                <li><img src="/static/image/pictures/papers_20230228.008.jpg" alt="picture" width="800" height="400"></li>
                
                <li><img src="/static/image/pictures/papers_20230228.007.jpg" alt="picture" width="800" height="400"></li>
                
                <li><img src="/static/image/pictures/papers_20230228.006.jpg" alt="picture" width="800" height="400"></li>
                
                <li><img src="/static/image/pictures/papers_20230228.005.jpg" alt="picture" width="800" height="400"></li>
                
                <li><img src="/static/image/pictures/papers_20230228.004.jpg" alt="picture" width="800" height="400"></li>
                
                <li><img src="/static/image/pictures/papers_20230228.003.jpg" alt="picture" width="800" height="400"></li>
                
                <li><img src="/static/image/pictures/papers_20230228.002.jpg" alt="picture" width="800" height="400"></li>
                
                <li><img src="/static/image/pictures/papers_20230228.001.jpg" alt="picture" width="800" height="400"></li>
                
            </ul>
            <a href="#" class="uk-slidenav uk-slidenav-contrast uk-slidenav-previous" data-uk-slideshow-item="previous" style="color: rgba(255, 0, 136, 0.6)"></a>
            <a href="#" class="uk-slidenav uk-slidenav-contrast uk-slidenav-next" data-uk-slideshow-item="next" style="color: rgba(255, 0, 136, 0.6)"></a>
        </div>
    </div>
    <div style="float:left;width:38%;margin:14px 0 0 35px;">
        <!--    <div class="uk-width-medium-3-5">.  -->
            <div class="uk-panel uk-panel-box uk-panel-box-secondary uk-panel-header" style="height:302px;overflow-y:auto;">    <!--style="height: 520px;"  -->
                <h2 class="uk-panel-title">News</h2>
                <ul class="uk-list-space uk-vertical-align-top">
                    
                    <li>
                        <p class="uk-article-lead"><em class="uk-text-danger">2025 October</em>: Si-Nan, Yu-Yao, Mo-Han, Zhi-Can, Xu-Kai and Li have published an article in Nucleic Acids Research. Well done!</p>
                    </li>
                    
                    <li>
                        <p class="uk-article-lead"><em class="uk-text-danger">2025 September</em>: Mei, Guo-Hua, Jia, Yi-Lin and Li have contributed an article in Nature communications. Congratulations!</p>
                    </li>
                    
                    <li>
                        <p class="uk-article-lead"><em class="uk-text-danger">2025 September</em>: Pei-Hong, Hua, Xu-Kai, Fang and Li have contributed an article in Genomics, Proteomics & Bioinformatics. Congratulations!</p>
                    </li>
                    
                </ul>
                <a class="uk-button-link uk-align-right uk-text-large" href="/news" type="button">More</a>
            </div>
       <!--   </div>   -->

    </div>
</div>

<div id="modal-center-fig1" class="uk-modal">
    <div class="uk-modal-dialog uk-modal-dialog-large">

    <a class="uk-modal-close uk-close"></a>

    <div class="uk-grid">
        <div class="uk-width-1-2">
        <img src="/static/image/research/research_fig1.jpg" alt="picture" >
        <p style="text-align:center;"><b>Fig 1.</b> Computational pipelines and biology of circRNAs.</p>
        </div>
        <div class="uk-width-1-2">
        <p style="font-size:16px"><b>1. Genome-wide profiling and characterization of functional circRNAs</b><br>Although only about 2% of the human genome encode protein sequences, recent advances in genome-wide analyses have revealed that the majority of the human genome are transcribed, largely from noncoding segments that were used to be considered as “junk sequences” or “dark matter” of the human genome. Besides well-characterized housekeeping noncoding RNAs (ncRNAs, such as tRNA, rRNA, snRNA and snoRNA) and small regulatory ncRNAs, the transcriptome has become even more complex with pervasively transcribed lncRNAs (at least 200 nt long without coding potential). By collaborating with experimental biologists, we have pioneered in developing poly(A)– RNA-seq and revealed a series of novel types of lncRNAs with specific structural features, including snoRNA-related lncRNAs (sno-lncRNAs; Yin <i>et al</i> 2012 <b><i>Mol Cell</i></b>; Zhang <i>et al</i> 2014 <b><i>BMC Genomics</i></b>), circular intronic RNAs (ciRNAs; Zhang <i>et al</i> 2013 <b><i>Mol Cell</i></b>), 5’ snoRNA-capped and 3’-polyadenylated lncRNAs (SPAs; Wu <i>et al</i> 2016 <b><i>Mol Cell</i></b>) and covalently-closed circRNAs produced from back-splicing of exons (circRNAs; Zhang <i>et al</i> 2014 <b><i>Cell</i></b>). Here, I mainly summarize systematic findings and our current understanding of circRNA biogenesis and functions, identified in my lab with collaboration (Fig.1). </p>
        </div>
    </div>

    </div>
</div>
<div id="modal-center-fig2" class="uk-modal">
    <div class="uk-modal-dialog uk-modal-dialog-large">

    <a class="uk-modal-close uk-close"></a>

    <div class="uk-grid">
        <div class="uk-width-1-2">
        <img src="/static/image/research/research_fig2.jpg" alt="picture" >
        <p style="text-align:center;"><b>Fig 2.</b> From editing biology to editing technology.</p>
        </div>
        <div class="uk-width-1-2">
        <p style="font-size:16px"><b>2. Nucleotide editing modification and genome base editing technologies</b><br>Biochemical editing/modification on DNA/RNA not only expands the number of functions encoded by our genomes but also provides additional mechanisms of gene regulation. Over 100 different types of RNA editing/modifications have been identified transcriptome-wide. The most predominant form of such editing/modifications in higher eukaryotes is adenosine-to-inosine (A-to-I) editing on RNA, catalyzed by members of adenosine deaminases acting on RNA (ADAR) enzyme family. As the resulting I preferentially bases pair with cytidine (C) and is therefore functionally guanosine (G), A-to-I editing can have profound effects on downstream RNA processing and function, including recoding of open reading frames, altering the pattern of alternative splicing, interfering with microRNA function, modulating RNAi activity, and playing other roles in gene regulation. In addition to adenosine deaminases, cytidine deaminases are also known to be responsible for cytidine deamination on both single strand DNA (ssDNA) and RNA, catalyze by the apolipoprotein B mRNA editing enzyme, catalytic polypeptide-like/activation-induced cytidine deaminase (APOBEC/AID) family of enzymes. Another major focus of my lab is developing new computational pipelines to globally profile nucleobase editing/modification events and to inspect potential crosstalk between different types of editing/modifications (Xiang <i>et al</i> 2018 <b><i>Mol Cell</i></b>). Importantly, by underscoring a previously underappreciated linkage between APOBECs with clustered regularly interspaced short palindromic repeat (CRISPR)-CRISPR associated system (Cas) 9-generated DNA breaks, my lab has contributed in the genome editing field by generating a variety of base editing systems to perform C-to-T changes at target genomic sites in broad genome backgrounds (Lei <i>et al</i> 2018 <b><i>Nat Struct Mol Biol</i></b>; Li <i>et al</i> 2018 <b><i>Nat Biotechnol</i></b>; Wang <i>et al</i> 2018 <b><i>Nat Biotechnol</i></b>; Wang <i>et al</i> 2019 <b><i>Genome Biol</i></b>; Wang <i>et al</i> 2020 <b><i>Cell Rep</i></b>) (Fig. 2).</p>
        </div>
    </div>

    </div>
</div>

<div style="clear:left;height:50px"></div>

<div class="uk-grid" data-uk-grid-margin>

    <div class="uk-width-medium-3-5">
        <div class="uk-panel uk-panel-box uk-panel-box-secondary uk-panel-header" style="height:590px;">
            <h2 class="uk-panel-title">Research Summary</h2>
            <div class="uk-width-1-1">
                <p>The completion of the human genome project (HGP) in the beginning of this century and the application of affordable high-throughput sequencing technologies in the past decade have led life science researches to the post-genome era. The resulting wealth of deep-sequencing datasets at genome (including epi-genome) and transcriptome (including epi-transcriptome) levels challenges us to fully understand how functional genomic elements are transcribed and regulated, thus leading to human health and/or diseases. Importantly, the advent of novel genome editing technologies provides us powerful methods to change genetic information at desired target sites at single nucleotide resolution, which benefits not only basic research aiming to decipher how different genotypes result in distinct phenotypes but also pre-clinical study potential to cure human diseases caused by genetic mutations.</p>
<p>Since established in 2011, my lab has continuously developed and applied computational strategies, including the-state-of-the-art machine learning and deep learning approaches, together with deep-sequencing technologies and diverse biochemical methods, <ul style="margin-top:-10px;"><li>to dissect complex expression, novel function and potential application of long noncoding RNAs (lncRNAs) and circular RNAs (circRNAs) from back-splicing;</li><li>to achieve precise genetic changes in the application of biomedical research and therapeutics by the invention of novel genome base editing systems;</li><li>to uncover previously-underappreciated modes of RNA regulation, such as alternative polyadenylation and subcellular localization, at single-cell resolution.</li></ul></p>
<h3>Research Achievements</h3>
<p><a href="#modal-center-fig1" data-uk-modal="{center:true}">1. Genome-wide profiling and characterization of functional circRNAs</a><br><a href="#modal-center-fig2" data-uk-modal="{center:true}">2. Nucleotide editing modification and genome base editing technologies</a><br>3. Novel machine/deep learning approaches for the understanding of RNA regulation</p>
            </div>
            <!-- <a class="uk-button-link uk-align-right uk-text-large" href="/news" type="button">More</a> -->
        </div>
    </div>

    <div class="uk-width-medium-2-5">

        <div class="uk-panel uk-panel-box uk-panel-box uk-panel-box-secondary uk-panel-header">
            <h2 class="uk-panel-title">Collaborators</h2> 
            
            <p><a class="uk-link-muted" href="http://chenlab-rna.sibcb.ac.cn"><i class="uk-icon-user"></i> Prof. Dr. Ling-Ling Chen, SIBCB, SIBS, CAS</a></p>
            
            <p><a class="uk-link-muted" href="http://facultydirectory.uchc.edu/profile?profileId=Carmichael-Gordon"><i class="uk-icon-user"></i> Prof. Dr. Gordon G. Carmichael, UConn Health</a></p>
            
            <p><a class="uk-link-muted" href="http://slst.shanghaitech.edu.cn/english/2017/1110/c730a4928/page.htm"><i class="uk-icon-user"></i> Prof. Dr. Jia Chen, ShanghaiTech University</a></p>
            
        </div>

        <div class="uk-panel uk-panel-box uk-panel-box uk-panel-box-secondary uk-panel-header" style="margin-top: 18px">
            <h2 class="uk-panel-title">Links</h2>
            
            <p><a class="uk-link-muted" href="https://github.com/YangLab"><i class="uk-icon-link"></i> GitHub</a></p>
            
            <p><a class="uk-link-muted" href="https://www.researchgate.net/profile/Li_Yang25"><i class="uk-icon-link"></i> ResearchGate</a></p>
            
            <p><a class="uk-link-muted" href="https://scholar.google.com/citations?user=gXlNLREAAAAJ"><i class="uk-icon-link"></i> Google Scholar</a></p>
            
            <p><a class="uk-link-muted" href="http://www.picb.ac.cn/rnomics/labmanagerx"><i class="uk-icon-link"></i> LabManagerX</a></p>
            
        </div>

        <div class="uk-panel uk-panel-box uk-panel-box uk-panel-box-secondary uk-panel-header" style="margin-top: 18px">
            <h2 class="uk-panel-title">Contact</h2>
            <p><i class="uk-icon-street-view"></i> Address: 科研二号楼CB2-044-046, 131 Dong An Road, Shanghai 200031, P.R.China</p>
            <p><i class="uk-icon-phone"></i> Telephone: (+86)-021-54661081</p>
            <p><i class="uk-icon-envelope"></i> Email: liyang_fudan@fudan.edu.cn</p>
        </div>

    </div>
    
</div>




        </article>

        
<hr class="uk-grid-divider">
<div class="uk-panel uk-text-center">
    <script type="text/javascript" src="//rc.revolvermaps.com/0/0/3.js?i=2lkeqnxkw2h&amp;b=0&amp;s=30&amp;m=2&amp;cl=ffffff&amp;co=010020&amp;cd=aa0000&amp;v0=60&amp;v1=60&amp;r=1" async="async"></script>
    <p>©Copyright 2012-2024 Bioinformatics and Interdisciplinary Technologies (BITs), Yang Lab. All rights reserved.</p>
</div>


    </div>

    <div id="nav-offcanvas" class="uk-offcanvas">
    <div class="uk-offcanvas-bar">
        <ul class="uk-nav uk-nav-offcanvas" data-uk-nav>
            <li><a href="/index"><i class="uk-icon-home"></i> Home</a></li>
            <li><a href="/news"><i class="uk-icon-newspaper-o"></i> News</a></li>
            <li><a href="/awards"><i class="uk-icon-newspaper-o"></i> Awards</a></li>
            <li><a href="/pictures"><i class="uk-icon-picture-o"></i> Pictures</a></li>
            <li><a href="/team"><i class="uk-icon-users"></i> Team</a></li>
            <li><a href="/publications"><i class="uk-icon-file-text"></i> Publications</a></li>
            <li><a href="/resources"><i class="uk-icon-wrench"></i> Resources</a></li>
            <!--<li><a href="/SM"><i class="uk-icon-cubes"></i> Supplementary Materials</a></li>-->
            <li><a href="/login"><i class="uk-icon-user"></i> Login</a></li>
        </ul>
    </div>
</div>

    <script type="text/javascript">
    $('.uk-navbar a:contains("Home")').parent().addClass('uk-active');
    $('.uk-nav a:contains("Home")').parent().addClass('uk-active');
    </script>
    <!-- Piwik -->
    <script type="text/javascript">
    var _paq = _paq || [];
    _paq.push(['trackPageView']);
    _paq.push(['enableLinkTracking']);
    (function() {
        var u="//www.picb.ac.cn/rnomics/piwik/";
        _paq.push(['setTrackerUrl', u+'piwik.php']);
        _paq.push(['setSiteId', 1]);
        var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
        g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
    })();
    </script>
    <noscript><p><img src="//www.picb.ac.cn/rnomics/piwik/piwik.php?idsite=1" style="border:0;" alt="" /></p></noscript>
    <!-- End Piwik Code -->

    

</body>

</html>