<!DOCTYPE html>

<html lang="en">


<head>
    <meta charset="UTF-8">
    <meta name="description" content="Publications of Bioinformatics and Interdisciplinary Technologies (BITs)">
    <meta name="author" content="Bioinformatics and Interdisciplinary Technologies (BITs)">
    <link rel="shortcut icon" href="/static/image/favicon.ico">
    <title>Bioinformatics and Interdisciplinary Technologies (BITs) &mdash; Yang Lab</title>

    
    <link rel="stylesheet" href="/static/css/uikit.almost-flat.min.css" type="text/css">
    <link rel="stylesheet" href="/static/css/components/sticky.almost-flat.min.css" type="text/css">
    <script src="/static/js/jquery.min.js" type="text/javascript" charset="utf-8"></script>
    <script src="/static/js/uikit.min.js" type="text/javascript" charset="utf-8"></script>
    <script src="/static/js/components/sticky.min.js" type="text/javascript" charset="utf-8"></script>
    <script src="/static/js/components/grid.min.js" type="text/javascript" charset="utf-8"></script>
    

</head>

<body>

    

    <div class="uk-container uk-container-center uk-margin-large-top uk-margin-large-bottom">

        <div class="uk-grid" data-uk-grid-margin>
    <div class="uk-width-1-1">
        <!--<h1><img src="/static/image/favicon.png" alt="logo">Computational Biology and Biotechnologies (Yang Lab) <br> <text>on genomics, transcriptomics and genome editing<text></h1>-->
        <span>
        <div style="display:inline-block"><img src="/static/image/favicon.png" alt="logo"></div>
        <div style="display:inline-block">  
            <span style="font-size: 36px">Bioinformatics and Interdisciplinary Technologies (BITs), Yang Lab</span>
            <br>
        </div>
        </span>
    </div>
</div>

        <nav class="uk-navbar uk-margin-top uk-margin-bottom" data-uk-sticky="{top:-50}">
    <a class="uk-navbar-brand uk-hidden-small" href="#">Yang Lab</a>
    <ul class="uk-navbar-nav uk-hidden-small">
        <li><a href="/index"><i class="uk-icon-home"></i> Home</a></li>
       <!-- <li><a href="/research"><i class="uk-icon-laptop"></i> Research</a></li>  -->
        <li><a href="/news"><i class="uk-icon-newspaper-o"></i> News</a></li>
        
        <li><a href="/team"><i class="uk-icon-users"></i> Team</a></li>
        <li><a href="/publications"><i class="uk-icon-file-text"></i> Publications</a></li>

        <li><a href="/awards"><i class="uk-icon-star"></i> Awards</a></li>
        <li><a href="/pictures"><i class="uk-icon-picture-o"></i> Pictures</a></li>

        <li><a href="/resources"><i class="uk-icon-wrench"></i> Resources</a></li>
        <!--<li><a href="/SM"><i class="uk-icon-cubes"></i> Supplementary Materials</a></li>-->
    </ul>
    <a href="#nav-offcanvas" class="uk-navbar-toggle uk-visible-small" data-uk-offcanvas></a>
    <div class="uk-navbar-brand uk-navbar-center uk-visible-small">Yang Lab</div>
</nav>

        <article class="uk-article">

            
<div class="uk-grid" data-uk-grid-margin>
    <div class="uk-width-1-1">
        <h2 class="uk-article-title">Publications</h2>
    </div>
</div>

<div class="uk-grid" data-uk-grid-margin>

    <div class="uk-width-medium-3-4">
        
        <a id="2023" style="position: relative;top: -50px;"></a>
        <h2>2023:</h2>
        <ul class="uk-list-space">
            
            <li>Vromman M, Anckaert J, Bortoluzzi S, Buratin A, Chen CY, Chu Q, Chuang TJ, Dehghannasiri R, Dieterich C, Dong X, Flicek P, Gaffo E, Gu W, He C, Hoffmann S, Izuogu O, Jackson MS, Jakobi T, Lai EC, Nuytens J, Salzman J, Santibanez-Koref M, Stadler P, Thas O, Vanden Eynde E, Verniers K, Wen G, Westholm J, <strong>Yang L</strong>, Ye CY, Yigit N, Yuan GH, Zhang J, Zhao F, Vandesompele J and Volders PJ. Large-scale benchmarking of circRNA detection tools reveals large differences in sensitivity but not in precision. <strong>Nat Methods</strong>, 2023,  10.1038/s41592-023-01944-6 
                <a href="https://www.nature.com/articles/s41592-023-01944-6"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2023NMethods_circRNA_benchmark.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Ortiz-Barahona V, Soler M, Davalos V, Garcia-Prieto CA, Janin M, Setien F, Fernandez-Rebollo I, Bech-Serra JJ, De La Torre C, Guil S, Villanueva A, Zhang PH, <strong>Yang L</strong>, Guarnacci M, Schumann U, Preiss T, Balaseviciute U, Montal R, Llovet JM and Esteller M*. Epigenetic inactivation of the 5-methylcytosine RNA methyltransferase NSUN7 is associated with clinical outcome and therapeutic vulnerability in liver cancer. <strong>Mol Cancer</strong>, 2023, 22(1): 83 
                <a href="https://molecular-cancer.biomedcentral.com/articles/10.1186/s12943-023-01785-z"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2023MolCancer_NSUN7.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Shan L#, Xu G#, Yao RW, Luan PF, Huang Y, Zhang PH, Pan YH, Zhang L, Gao X, Li Y, Cao SM, Gao SX, Yang ZH, Li S, Yang LZ, Wang Y, Wong CCL, Yu L, Li J, <strong>Yang L</strong> and Chen LL*. Nucleolar URB1 ensures 3' ETS rRNA removal to prevent exosome surveillance. <strong>Nature</strong>, 2023, 615(7952): 526-534 
                <a href="https://www.nature.com/articles/s41586-023-05767-5"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2023_Nature_Nucleolar%20URB1%20ensures%203%20ETS%20rRNA%20removal%20to%20prevent%20exosome%20surveillance.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wang Y, Qi T, Liu J, Yang Y, Wang Z, Wang Y, Wang T, Li M, Li M, Lu D, Chang ACY, <strong>Yang L</strong>, Gao S, Wang Y* and Lan F*. A highly specific CRISPR-Cas12j nuclease enables allele-specific genome editing. <strong>Sci Adv</strong>, 2023, 9(6): eabo6405 
                <a href="https://www.science.org/doi/10.1126/sciadv.abo6405"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2023_SciAdv_Cas12j.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Huang Y, Gao BQ, Meng Q, Yang LZ, Ma XK, Wu H, Pan YH, <strong>Yang L</strong>, Li D and Chen LL*. CRISPR-dCas13-tracing reveals transcriptional memory and limited mRNA export in developing zebrafish embryos. <strong>Genome Biol</strong>, 2023, 24(1): 15 
                <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02848-6"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2023_GenomeBiol_CRISPR-dCas13-tracing_reveals_transcriptional_memory_and_limited_mRNA_export.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Chen LL*, Bindereif A, Bozzoni I, Chang HY, Matera AG, Gorospe M, Hansen TB, Kjems J, Ma XK, Pek JW, Rajewsky N, Salzman J, Wilusz JE*, <strong>Yang L</strong>* and Zhao F. A guide to naming eukaryotic circular RNAs. <strong>Nat Cell Biol</strong>, 2023, 25(1): 1-5 
                <a href="https://www.nature.com/articles/s41556-022-01066-9"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2023_NatCellBiol_A_guide_to_naming_eukaryotic_circular_RNAs.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wang B*#, Jiang B#, Li GW#, Dong F, Luo Z, Cai B, Wei M, Huang J, Wang K, Feng X, Tong F, Wang S, Wang Q, Han Q, Li C, Zhang X, <strong>Yang L</strong>* and Bao L*. Somatosensory neurons express specific sets of lincRNAs, and lincRNA CLAP promotes itch sensation in mice. <strong>EMBO Rep</strong>, 2023, 24(2): e54313 
                <a href="https://www.embopress.org/doi/abs/10.15252/embr.202154313"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2022-EMBO_Rep-Somatosensory_neurons_lincRNAs_and_itch.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Yuan GH#, Wang Y#, Wang GZ and <strong>Yang L</strong>*. RNAlight: a machine learning model to identify nucleotide features determining RNA subcellular localization. <strong>Brief Bioinform</strong>, 2023, 24(1): bbac509 
                <a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbac509/6868526"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2023_BriefBioinformatics_RNAlight.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
        </ul>
        
        <a id="2022" style="position: relative;top: -50px;"></a>
        <h2>2022:</h2>
        <ul class="uk-list-space">
            
            <li>Huang J, Jiang B, Li GW, Zheng D, Li M, Xie X, Pan Y, Wei M, Liu X, Jiang X, Zhang X, <strong>Yang L</strong>, Bao L* and Wang B*. m(6)A-modified lincRNA Dubr is required for neuronal development by stabilizing YTHDF1/3 and facilitating mRNA translation. <strong>Cell Rep</strong>, 2022, 41(8): 111693 
                <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(22)01567-4"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2022_Huang_CellReports_m6A-modified%20lincRNA%20Dubr.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Gopalan V, Nilsen T, Gopalan V, Altman AM, Stark BC, Feinstein SI, Koski R, Mickiewicz C, Stark B, Gegenheimer P, Kirsebom LA, Arnez JG, Forster AC, Kazakov SA, Yuan Y, Liu F, Jarrous N, <strong>Yang L</strong>, Jiang G, Jiang T, Rosenbaum JL, Miller G, DiMaio D, Carlson JR, McClain WH, Mathews MB, Kaempfer R, Deutscher MP, Chen LL, Li Y, Wang E, Patutina O, Zenkova M, Vlassov V, Lucks JB and Gopalan V. Tribute to Sidney Altman. <strong>RNA</strong>, 2022, 28(11): 1393-1429 
                <a href="https://doi.org/10.1261/rna.079397.122"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2022RNA_tribute2Sid_li.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Yang LZ#, Gao BQ#, Huang Y, Wang Y, <strong>Yang L</strong> and Chen LL*. Multi-color RNA imaging with CRISPR-Cas13b systems in living cells. <strong>Cell Insight</strong>, 2022, 1(4):100044 
                <a href="https://doi.org/10.1016/j.cellin.2022.100044"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2022_CellInsight.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li><strong>Yang L</strong>*, Wilusz JE* and Chen LL*. Biogenesis and Regulatory Roles of Circular RNAs. <strong>Annu Rev Cell Dev Biol</strong>, 2022, 38263-289 
                <a href="https://www.annualreviews.org/doi/10.1146/annurev-cellbio-120420-125117"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2022_Annurev_Biogenesis_and_Regulatory_Roles_of_Circular_RNAs.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Li X#, Zhou L#, Gao BQ#, Li G, Wang X, Wang Y, Wei J, Han W, Wang Z, Li J, Gao R, Zhu J, Xu W, Wu J, Yang B, Sun X*, <strong>Yang L</strong>* and Chen J*. Highly efficient prime editing by introducing same-sense mutations in pegRNA or stabilizing its structure. <strong>Nat Commun</strong>, 2022, 13(1): 1669 
                <a href="https://www.nature.com/articles/s41467-022-29339-9"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2022NC_PE_spegRNA_apegRNA.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Gao R#, Fu ZC#, Li X#, Wang Y#, Wei J, Li G, Wang L, Wu J, Huang X*, <strong>Yang L</strong>* and Chen J*. Genomic and Transcriptomic Analyses of Prime Editing Guide RNA-Independent Off-Target Effects by Prime Editors. <strong>CRISPR J</strong>, 2022, 5(2): 276-293 
                <a href="https://www.liebertpub.com/doi/10.1089/crispr.2021.0080"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2022_CRISPRJ_PE_Off-Target.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Gao X#, Ma XK#, Li X, Li GW, Liu CX, Zhang J, Wang Y, Wei J, Chen J, Chen LL and <strong>Yang L</strong>*. Knockout of circRNAs by base editing back-splice sites of circularized exons. <strong>Genome Biol</strong>, 2022, 23(1): 16 
                <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02563-0"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2022GB_BEable_circRNA.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Liu CX#, Guo SK#, Nan F, Xu YF, <strong>Yang L</strong> and Chen LL*. RNA circles with minimized immunogenicity as potent PKR inhibitors. <strong>Mol Cell</strong>, 2022, 82(2): 420-434 e426 
                <a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(21)01009-1"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2022-Mol_Cell-circRNA_PKR_inhibitors.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
        </ul>
        
        <a id="2021" style="position: relative;top: -50px;"></a>
        <h2>2021:</h2>
        <ul class="uk-list-space">
            
            <li>Zhang Y, <strong>Yang L</strong>* and Chen LL*. Characterization of Circular RNAs. <strong>Methods Mol Biol</strong>, 2021, 2372: 179-192 (Book chapter) 
                <a href="https://link.springer.com/protocol/10.1007/978-1-0716-1697-0_16"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2021-Methods_Mol_Biol-circular_RNAs-Book_chapter.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Li X, Zhang JL, Lei YN, Liu XQ, Xue W, Zhang Y, Nan F, Gao X, Zhang J, Wei J, <strong>Yang L</strong> and Chen LL*. Linking circular intronic RNA degradation and function in transcription by RNase H1. <strong>Sci China Life Sci</strong>, 2021, 64(11): 1795-1809 
                <a href="https://link.springer.com/article/10.1007/s11427-021-1993-6"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2021_Sci%20China%20Life%20Sci_LinkingCircularIntronicRNADegr.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Li GW#, Nan F#, Yuan GH, Liu CX, Liu X, Chen LL, Tian B and <strong>Yang L</strong>*. SCAPTURE: a deep learning-embedded pipeline that captures polyadenylation information from 3' tag-based RNA-seq of single cells. <strong>Genome Biol</strong>, 2021, 22(1): 221 
                <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02437-5"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2021_GenomeBiology_SCAPTUR.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wu M#, Xu G#, Han C#, Luan PF, Xing YH, Nan F, Yang LZ, Huang Y, Yang ZH, Shan L, <strong>Yang L</strong>, Liu J* and Chen LL*. lncRNA SLERT controls phase separation of FC/DFCs to facilitate Pol I transcription. <strong>Science</strong>, 2021, 373(6554): 547-555 
                <a href="https://science.sciencemag.org/content/373/6554/547"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2021_Science_lncRNA%20SLERT%20controls%20phase%20separation.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Highlighted by <strong>Science</strong>, 2021, 373: 486-487</li>
                
            </ul>
             
            <li>Wang L#, Xue W#, Zhang H#, Gao R#, Qiu H#, Wei J, Zhou L, Lei YN, Wu X, Li X, Liu C, Wu J, Chen Q, Ma H, Huang X, Cai C, Zhang Y, Yang B*, Yin H*, <strong>Yang L*</strong> and Chen J*. Eliminating base-editor-induced genome-wide and transcriptome-wide off-target mutations. <strong>Nat Cell Biol</strong>, 2021, 23(5): 552-563 
                <a href="https://www.nature.com/articles/s41556-021-00671-4"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2021_NatCellBiol_tBE_BEIDOU.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wei M#, Huang J#, Li GW, Jiang B, Cheng H, Liu X, Jiang X, Zhang X, <strong>Yang L</strong>, Bao L* and Wang B*. Axon-enriched lincRNA ALAE is required for axon elongation via regulation of local mRNA translation. <strong>Cell Rep</strong>, 2021, 35(5): 109053 
                <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(21)00369-7"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2021-CellReports-Axon-enriched%20lincRNA%20ALAE%20is%20required%20for%20axon%20elongation.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Ma XK, Xue W, Chen LL and <strong>Yang L</strong>*. CIRCexplorer pipelines for circRNA annotation and quantification from non-polyadenylated RNA-seq datasets. <strong>Methods</strong>, 2021, 1963-10 
                <a href="https://www.sciencedirect.com/science/article/pii/S1046202321000451"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2021_Methods_CIRCexplorer%20pipelines.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Guo SK#, Nan F#, Liu CX, <strong>Yang L</strong> and Chen LL*. Mapping circular RNA structures in living cells by SHAPE-MaP. <strong>Methods</strong>, 2021, 19647-55 
                <a href="https://www.sciencedirect.com/science/article/pii/S1046202321000232"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2021_Methods_SHAPE-MaP.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Xue W, Ma XK and <strong>Yang L*</strong>. Fast and Furious: insights of back splicing regulation during nascent RNA synthesis. <strong>Sci China Life Sci</strong>, 2021, 64(7): 1050-1061 
                <a href="http://engine.scichina.com/doi/10.1007/s11427-020-1881-1"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2021_Sci.%20China%20Life%20Sci._Fast%20and%20Furious.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Li S#, Li X#, Xue W#, Zhang L#, Yang LZ, Cao SM, Lei YN, Liu CX, Guo SK, Shan L, Wu M, Tao X, Zhang JL, Gao X, Zhang J, Wei J, Li J*, <strong>Yang L</strong>* and Chen LL*. Screening for functional circular RNAs using the CRISPR-Cas13 system. <strong>Nat Methods</strong>, 2021, 18(1): 51-59 
                <a href="https://www.nature.com/articles/s41592-020-01011-4"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2020_Nat.Methods_Screening_circRNA_Cas13.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Highlighted by <strong>Nat Rev Genet</strong>, 2021, 22: 68 <a href="https://www.nature.com/articles/s41576-020-00318-4"><i class="uk-icon-external-link"></i></a></li>
                
            </ul>
             
        </ul>
        
        <a id="2020" style="position: relative;top: -50px;"></a>
        <h2>2020:</h2>
        <ul class="uk-list-space">
            
            <li>Ma XK#, Guo CJ#, <strong>Yang L</strong> and Chen LL*. Altered processing of lncRNAs in stem cells contributes to non-conserved functions. <strong>Chemistry of Life</strong>, 2020, 40(7): 985-991 (Review, in Chinese) 
                <a href="http://dwz.win/W5M"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2020-Chemistry_of_life-Altered_processing_of_lncRNAs.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li><strong>Yang L*</strong> and Chen J*. A Tale of Two Moieties: Rapidly Evolving CRISPR/Cas-Based Genome Editing. <strong>Trends Biochem Sci</strong>, 2020, 45(10): 874-888 (Review) 
                <a href="https://www.sciencedirect.com/science/article/pii/S096800042030150X"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2020_TIBS_A%20Tale%20of%20Two%20Moieties.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wang X#, Ding C#, Yu W#, Wang Y#, He S#, Yang B#, Xiong Y-C, Wei J, Li J, Liang J, Lu Z, Zhu W, Wu J, Zhou Z, Huang X, Liu Z*, <strong>Yang L*</strong> and Chen J*. Cas12a Base Editors Induce Efficient and Specific Editing with Low DNA Damage Response. <strong>Cell Rep</strong>, 2020, 31(9):107723 
                <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(20)30700-2"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2020_CellRep_BEACON.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhang X#, Zhu B#, Chen L#, Xie L, Yu W, Wang Y, Li L, Yin S, Yang L, Hu H, Han H, Li Y, Wang L, Chen G, Ma X, Geng H, Huang W, Pang X, Yang Z, Wu Y, Siwko S, Kurita R, Nakamura Y, <strong>Yang L</strong>, Liu M and Li D*. Dual base editor catalyzes both cytosine and adenine base conversions in human cells. <strong>Nat Biotechnol</strong>, 2020, 38(7): 856-860 
                <a href="https://www.nature.com/articles/s41587-020-0527-y"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2020_NBT_Dual_base_editor_catalyzes_both_cytosine_and_adenine_base_conversions_in_human_cells.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Schumann U#, Zhang HN#, Sibbritt T, Pan A, Horvath A, Gross S, Clark SJ, <strong>Yang L</strong> and Preiss T*. Multiple links between 5-methylcytosine content of mRNA and translation. <strong>BMC Biol</strong>, 2020, 18(1):40 
                <a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-020-00769-5"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2020_BMC_Biology_Multiple_links_between_5-methylcytosine.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Guo CJ#, Ma XK#, Xing YH, Zheng CC, Xu YF, Shan L, Zhang J, Wang S, Wang Y, Carmichael GG, <strong>Yang L</strong> and Chen LL*. Distinct Processing of lncRNAs Contributes to Non-conserved Functions in Stem Cells. <strong>Cell</strong>, 2020, 181(3):621-636.e22 
                <a href="https://doi.org/10.1016/j.cell.2020.03.006"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2020_Cell_Distinct_Processing_of_lncRNAs.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Cai C#, Geng A#, Wang M, <strong>Yang L</strong>, Yu QC* and Zeng YA*. Amphiregulin mediates the hormonal regulation on Rspondin-1 expression in the mammary gland. <strong>Dev Biol</strong>, 2020, 458(1):43-51 
                <a href="https://www.sciencedirect.com/science/article/pii/S0012160619302325?via%3Dihub"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2020_Dev_Biol_Amphiregulin_Rspondin-1.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
        </ul>
        
        <a id="2019" style="position: relative;top: -50px;"></a>
        <h2>2019:</h2>
        <ul class="uk-list-space">
            
            <li>Ma XK, Wang MR, Liu CX, Dong R, Carmichael GG, Chen LL and <strong>Yang L*</strong>. CIRCexplorer3: A CLEAR pipeline for direct comparison of circular and linear RNA expression. <strong>Genomics Proteomics Bioinformatics</strong>, 2019, 17(5):511–521 
                <a href="https://www.sciencedirect.com/science/article/pii/S1672022919301664?via%3Dihub"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_GPB_CIRCexplorer3.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wang Y#, Gao R#, Wu J#, Xiong YC, Wei J, Zhang S, Yang B, Chen J* and <strong>Yang L*</strong>. Comparison of cytosine base editors and development of the BEable-GPS database for targeting pathogenic SNVs. <strong>Genome Biol</strong>, 2019, 20(1):218 
                <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1839-4"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_Genome_Res-BEable-GPS.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li><strong>Yang L*</strong>, Yang B* and Chen J*. One Prime for All Editing. <strong>Cell</strong>, 2019, 179(7): 1448-1450 (Preview) 
                <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)31287-5"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_Cell_One_Prime_for_All_Editing.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Yao RW#, Xu G#, Wang Y, Shan L, Luan PF, Wang Y, Wu M, Yang LZ, Xing YH, <strong>Yang L</strong> and Chen LL*. Nascent Pre-rRNA Sorting via Phase Separation Drives the Assembly of Dense Fibrillar Components in the Human Nucleolus. <strong>Mol Cell</strong>, 2019, 76(5): 1-17 
                <a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(19)30630-6"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_Mol_Cell_%20phase_separation_pre-rRNA_DFC.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhu YJ#, Zheng B#, Luo GJ#, Ma XK#, Lu XY, Lin XM, Yang S, Zhao Q, Wu T, Li ZX, Liu XL, Wu R, Liu JF, Ge Y, <strong>Yang L</strong>, Wang HY* and Chen L*. Circular RNAs negatively regulate cancer stem cells by physically binding FMRP against CCAR1 complex in hepatocellular carcinoma. <strong>Theranostics</strong>. 2019, 9(12):3526-3540 
                <a href="http://www.thno.org/v09p3526.htm"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_Theranostics-circRNA_binding_FMRP_against_CCAR1.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Liu CX#, Li X#, Nan F#, Jiang S, Gao X, Guo SK, Xue W, Cui Y, Dong K, Ding H, Qu B, Zhou Z, Shen N*, <strong>Yang L*</strong> and Chen LL*. Structure and Degradation of Circular RNAs Regulate PKR Activation in Innate Immunity. <strong>Cell</strong>, 2019, 177(4): 865-880.e21 (<strong>Featured Article</strong>) 
                <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)30347-2?rss=yes"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_Cell-circRNAs_PKR.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Highlighted by <strong>Cell</strong>, 2019, 177: 797-799</li>
                
                <li>Highlighted by <strong>Science</strong>, 2019 , 364: 847</li>
                
                <li>Highlighted by <strong>Nat Rev Immunol</strong>, 2019, 19: 351</li>
                
                <li>Highlighted by <strong>Nat Rev Mol Cell Biol</strong>, 2019, 20: 387</li>
                
                <li>Highlighted by <strong>Cell Biosci</strong>, 2019, 9: 43</li>
                
                <li>Highlighted by <strong>F1000 Prime</strong>  <a href="https://f1000.com/prime/735608202"><i class="uk-icon-external-link"></i></a></li>
                
                <li>Highlighted by <strong>Science News Magazine</strong>  <a href="https://www.sciencenews.org/article/lack-circular-rna-may-trigger-lupus"><i class="uk-icon-external-link"></i></a></li>
                
            </ul>
             
            <li>Yang B*, <strong>Yang L*</strong> and Chen J*. Development and Application of Base Editors. <strong>CRISPR J</strong>, 2019(2), 2: 91-104 (Review) 
                <a href="https://www.liebertpub.com/doi/10.1089/crispr.2019.0001"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_CRISPR_J-BE_review.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Chen J*, Yang B* and <strong>Yang L*</strong>. To BE or not to BE, that is the question. <strong>Nat Biotechnol</strong>, 2019, 37(5): 520–522 (News & Views ) 
                <a href="https://www.nature.com/articles/s41587-019-0119-x"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_NBT_BE_news_and_views.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wang L#, Wang X#, <strong>Yang L</strong> and Chen J*. Development and application of base editor. <strong>Chemistry of Life</strong>, 2019, 39(1):13-20 (Review) 
                <a href="http://www.life.ac.cn/artsmore.asp?id=1301"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_ChemistryLife_Development_application_BE.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wang Y#, Xiong Y#, Chen J and <strong>Yang L*</strong>. Computational analysis in CRISPR/Cas genome editing. <strong>Chemistry of Life</strong>, 2019, 39(1):28-38 (Review) 
                <a href="http://www.life.ac.cn/artsmore.asp?id=1303"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_ChemistryLife_Computational_analysis_in_genome_editing.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Dong R, Ma XK, Chen LL and <strong>Yang L</strong>. Genome-Wide Annotation of circRNAs and Their Alternative Back-Splicing/Splicing with CIRCexplorer Pipeline. <strong>Methods Mol Biol</strong>, 2019, 1870: 137-149 (Book chapter) 
                <a href="https://link.springer.com/protocol/10.1007/978-1-4939-8808-2_10"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2019_MethodsMolBiol-CIRCexplorer_Pipeline.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
        </ul>
        
        <a id="2018" style="position: relative;top: -50px;"></a>
        <h2>2018:</h2>
        <ul class="uk-list-space">
            
            <li>Wang Y#, Hu SB#, Wang MR#, Yao RW, Wu D, <strong>Yang L</strong> and Chen LL*. Genome-wide screening of NEAT1 regulators reveals cross-regulation between paraspeckles and mitochondria. <strong>Nat Cell Biol</strong>, 2018, 20(10):1145-1158 
                <a href="https://www.nature.com/articles/s41556-018-0204-2"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2018_NatCellBiol_NEAT1.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Editorial by: Fox AH. <strong>Nat Cell Biol</strong>, 2018, 20(10): 1108-1109</li>
                
            </ul>
             
            <li>Dong R#, Ma XK#, Li GW and <strong>Yang L*</strong>. CIRCpedia v2: An Updated Database for Comprehensive Circular RNA Annotation and Expression Comparison. <strong>Genomics Proteomics Bioinformatics</strong>, 2018, 16(4):226-233 
                <a href="https://www.sciencedirect.com/science/article/pii/S1672022918302596"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2018_GPB_CIRCpediaV2.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wang X#, Li J#, Wang Y#, Yang B#, Wei J#, Wu J, Wang R, Huang X*, Chen J* and <strong>Yang L*</strong>. Efficient base editing in methylated regions with a human APOBEC3A-Cas9 fusion. <strong>Nat Biotechnol</strong>, 2018, 36(10):946-949 
                <a href="https://www.nature.com/articles/nbt.4198"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2018_NBT_APOBEC3A-Cas9%20.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhang R#, Gao Y#, Zhao X#, Gao M#, Wu Y, Han Y, Qiao Y, Luo Z, <strong>Yang L</strong>, Chen J and Ge G*. FSP1-positive fibroblasts are adipogenic niche and regulate adipose homeostasis. <strong>PLoS Biol</strong>, 2018, 16(8): e2001493 
                <a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2001493"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2018PLoSBiol.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Li X, <strong>Yang L*</strong> and Chen LL*. The Biogenesis, Functions, and Challenges of Circular RNAs. <strong>Mol Cell</strong>, 2018, 71(3):428-442 (Review) 
                <a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(18)30509-4"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2018MC_circRNA_review.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Li X#, Wang Y#, Liu Y#, Yang B#, Wang X, Wei J, Lu Z, Zhang Y, Wu J, Huang X*, <strong>Yang L*</strong> and Chen J*. Base editing with a cpf1-cytidine deaminase fusion. <strong>Nat Biotechnol</strong>, 2018, 36(4):324-327 
                <a href="https://www.nature.com/articles/nbt.4102"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2018_NBT_baseEditing.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Xiang JF#, Yang Q#, Liu CX#, Wu M, Chen LL* and <strong>Yang L*</strong>. N6-Methyladenosines Modulate A-to-I RNA Editing. <strong>Mol Cell</strong>, 2018, 69(1):126-135 
                <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30934-6"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2018_MC_m6A.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Lei L#, Chen H#, Xue W#, Yang B#, Hu B#, Wei J, Wang L, Cui Y, Li W, Wang J, Yan L, Shang W, Gao J, Sha J, Zhuang M, Huang X, Shen B*, <strong>Yang L*</strong> and Chen J*. APOBEC3 induces mutations during repair of CRISPR–Cas9-generated DNA breaks. <strong>Nat Struct Mol Biol</strong>, 2018, 25(1):45–52 
                <a href="https://www.nature.com/articles/s41594-017-0004-6"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2018_NSMB_APOBEC3.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
        </ul>
        
        <a id="2017" style="position: relative;top: -50px;"></a>
        <h2>2017:</h2>
        <ul class="uk-list-space">
            
            <li>Liang D, Tatomer DC, Luo Z, Wu H, <strong>Yang L</strong>, Chen LL, Cherry S, and Wilusz JE*. The Output of Protein-Coding Genes Shifts to Circular RNAs When the Pre-mRNA Processing Machinery Is Limiting. <strong>Mol Cell</strong>, 2017, 68(5):940–954 
                <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30835-3"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_protein_gene_circRNA.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li><strong>Yang L*</strong> and Chen LL*. Enhancing the RNA engineering toolkit. <strong>Science</strong>, 2017, 358(6366):996-997 (Perspectives) 
                <a href="http://science.sciencemag.org/content/358/6366/996.full"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_CRISPR_RNA_persperctive.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wang L#, Xue W#, Yan L#, Li X, Wei J, Chen M, Wu J, Yang B*, <strong>Yang L*</strong> and Chen J*. Enhanced base editing by co-expression of free uracil DNA glycosylase inhibitor. <strong>Cell Res</strong>, 2017, 27(10):1289-1292 
                <a href="http://www.nature.com/cr/journal/vaop/ncurrent/full/cr2017111a.html?foxtrotcallback=true"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_Cell_Res_enhanced_base_editing.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Yang Q, Wang Y and <strong>Yang L*</strong>. Multifaceted roles of complementary sequences on circRNA formation. <strong>Quantitative Biology</strong>, 2017, 5(3): 205-209 (Review) 
                <a href="https://link.springer.com/article/10.1007/s40484-017-0112-7"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_Quantitative_Biology_mini_review.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wu H, <strong>Yang L*</strong> and Chen LL*. The Diversity of Long Noncoding RNAs and Their Generation. <strong>Trends Genet</strong>, 2017, 33(8): 540-552 (Review) 
                <a href="http://www.sciencedirect.com/science/article/pii/S0168952517300859"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_Mol_Trends_Genet_Diverse_lncRNA.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Li X#, Liu CX#, Xue W#, Zhang Y, Jiang S, Yin QF, Wei J, Yao RW, <strong>Yang L*</strong> and Chen LL*.Coordinated circRNA Biogenesis and Function with NF90/NF110 in Viral Infection. <strong>Mol Cell</strong>, 2017, 67(2): 214-227 
                <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30363-5"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_Mol_Cell_circRNP_NF90_NF110.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Editorial by: Cadena C and Hur S. <strong>Mol Cell</strong>, 2017, 67: 163-164</li>
                
            </ul>
             
            <li>Xing YH#, Yao RW#, Zhang Y#, Guo CJ, Jiang S, Xu G, Dong R, <strong>Yang L</strong> and Chen LL*. SLERT Regulates DDX21 Rings Associated with Pol I Transcription. <strong>Cell</strong>, 2017, 169(4): 664-678.e16 
                <a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30424-5"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_cell_slert.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Chen LL* and <strong>Yang L*</strong>.  ALUternative Regulation for Gene Expression. <strong>Trends Cell Biol</strong>, 2017, 27(7): 480-490 (Review) 
                <a href="http://www.cell.com/trends/cell-biology/fulltext/S0962-8924(17)30002-8"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_Trends_Cell_Biol_ALUternative.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Niu N, Xiang JF, Yang Q, Wang L, Wei Z, Chen LL, <strong>Yang L</strong> and Zou W*. RNA-binding protein SAMD4 regulates skeleton development through translational inhibition of Mig6 expression. <strong>Cell Disc</strong>, 2017, 3: 16050 
                <a href="http://www.nature.com/articles/celldisc201650"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_Cell_Disc_skeleton.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Dong R#, Ma XK#, Chen LL* and <strong>Yang L*</strong>. Increased complexity of circRNA expression during species evolution. <strong>RNA Biol</strong>, 2017, 14(8): 1064-1074 
                <a href="http://www.tandfonline.com/doi/full/10.1080/15476286.2016.1269999	"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2017_RNA_Biol.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
        </ul>
        
        <a id="2016" style="position: relative;top: -50px;"></a>
        <h2>2016:</h2>
        <ul class="uk-list-space">
            
            <li>Wu H#, Yin QF#, Luo Z#, Yao RW, Zheng CC, Zhang J, Xiang JF, <strong>Yang L</strong> and Chen LL*. Unusual Processing Generates SPA LncRNAs that Sequester Multiple RNA Binding Proteins. <strong>Mol Cell</strong>, 2016, 64(3): 534-548 (<strong>Cover Article</strong>) 
                <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(16)30630-X"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2016_Mol_Cell_SPA.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Editorial by: Li R and Fox AH. <strong>Mol Cell</strong>, 2016, 64: 435-437</li>
                
            </ul>
             
            <li>Luo Z, Yang Q and <strong>Yang L*</strong>. RNA Structure Switches RBP Binding. <strong>Mol Cell</strong>, 2016, 64(2): 219-220 (Previews) 
                <a href="http://www.cell.com/molecular-cell/abstract/S1097-2765(16)30629-3"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2016_Mol_Cell_Structure.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhang XO#, Dong R#, Zhang Y#, Zhang JL, Luo Z, Zhang J, Chen LL* and <strong>Yang L*</strong>. Diverse alternative back-splicing and alternative splicing landscape of circular RNAs. <strong>Genome Res</strong>, 2016, 26(9): 1277-1287 
                <a href="http://genome.cshlp.org/content/26/9/1277.full.pdf+html"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2016_Genome_Res_splicing.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhang Y#, Xue W#, Li X, Zhang J, Chen S, Zhang JL, <strong>Yang L*</strong> and Chen LL*. The Biogenesis of Nascent Circular RNAs. <strong>Cell Rep</strong>, 2016, 15(3): 611-624 
                <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(16)30329-1?rss=yes"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2016_Cell_Rep_nascent.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Dong R, Zhang XO, Zhang Y, Ma XK, Chen LL and <strong>Yang L*</strong>. CircRNA-derived pseudogenes. <strong>Cell Res</strong>, 2016, 26(6): 747-750 
                <a href="http://www.nature.com/cr/journal/v26/n6/full/cr201642a.html"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2016_Cell_Res_pseudoGene.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhang Y, <strong>Yang L*</strong> and Chen LL*. Characterization of Circular RNAs. <strong>Methods Mol Biol</strong>, 2016, 1402: 215-227 (Book chapter) 
                <a href="http://link.springer.com/protocol/10.1007/978-1-4939-3378-5_17"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2016-Methods_Mol_Biol-circular_RNAs-Book_chapter.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhong C#, Xie Z#, Yin Q#, Dong R,  Yang S, Wu Y,  <strong>Yang L</strong> and Li J*. Parthenogenetic haploid embryonic stem cells efficiently support mouse generation by oocyte injection. <strong>Cell Res</strong>, 2016, 26(1): 131-134 
                <a href="http://www.nature.com/cr/journal/vaop/ncurrent/full/cr2015132a.html"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2016_Cell_Res_haploid.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
        </ul>
        
        <a id="2015" style="position: relative;top: -50px;"></a>
        <h2>2015:</h2>
        <ul class="uk-list-space">
            
            <li><strong>Yang L</strong>. Splicing noncoding RNAs from the inside out. <strong>WIREs RNA</strong>, 2015, 6(6): 651-660 (Review) 
                <a href="http://onlinelibrary.wiley.com/enhanced/doi/10.1002/wrna.1307"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-WIREs_RNA-Splicing_noncoding_RNAs_from_the_inside_out.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Brooks AN, Duff MO, May G, <strong>Yang L</strong>, Bolisetty M, Landolin J, Wan K, Sandler J, Celniker SE, Graveley BR* and Brenner SE*. Regulation of alternative splicing in Drosophila by 56 RNA binding proteins. <strong>Genome Res</strong>, 2015,  25(11): 1771-1780 
                <a href="http://genome.cshlp.org/content/early/2015/08/20/gr.192518.115?top=1"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-Genome_Res-Regulation_56_RBP.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Xiang JF, <strong>Yang L</strong> and Chen LL. The long noncoding RNA regulation at the MYC locus. <strong>Curr Opin Genet Dev</strong>, 2015, 33:41–48 (Review) 
                <a href="http://www.sciencedirect.com/science/article/pii/S0959437X15000763"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-Curr_Opin_Genet_Dev-MYC.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhong C#, Yin Q#, Xie Z#, Bai M#, Dong R#, Tang W, Xing YH, Zhang H, Yang S, Chen LL, Bartolomei MS, Ferguson-Smith A, Li D, <strong>Yang L*</strong>, Wu Y* and Li J*. CRISPR-Cas9-Mediated Genetic Screening in Mice with Haploid Embryonic Stem Cells Carrying a Guide RNA Library. <strong>Cell Stem Cell</strong>, 2015, 17(2): 221-232 
                <a href="http://www.sciencedirect.com/science/article/pii/S1934590915002659"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-Cell_Stem_Cell-Haploid_ESCs.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Chen LL* and <strong>Yang L*</strong>. Gear up in circles. <strong>Mol Cell</strong>, 2015, 58(5): 715-717 (Preview) 
                <a href="http://dx.doi.org/10.1016/j.molcel.2015.05.027"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-Mol_Cell-Gear_up_in_circles.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Hu SB, Xiang JF, Li X, Xu Y, Xue W, Huang M, Wong CC, Sagum CA, Bedford MT, <strong>Yang L</strong>, Cheng D, and Chen LL. Protein arginine methyltransferase CARM1 attenuates the paraspeckle-mediated nuclear retention of mRNAs containing IRAlus. <strong>Genes & Dev</strong>, 2015, 29(6): 630-645 
                <a href="http://genesdev.cshlp.org/content/29/6/630.abstract"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-Genes_Dev-CARM1.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Chen LL* and <strong>Yang L*</strong>. Regulation of circRNA biogenesis. <strong>RNA Biology</strong>, 2015, 12(4): 381-388 (Review) 
                <a href="http://www.tandfonline.com/doi/full/10.1080/15476286.2015.1020271#abstract"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-RNA_Biology-Regulation_of_circRNA_biogenesis.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Chen T#, Xiang JF#, Zhu S#, Chen S, Yin QF, Zhang XO, Zhang J, Feng H, Dong R, Li XJ, <strong>Yang L*</strong> and Chen LL*. 2015. ADAR1 is required for differentiation and neural induction by regulating microRNA processing in a catalytically independent manner. <strong>Cell Res</strong>, 2015, 25(4): 459-476 
                <a href="http://www.nature.com/cr/journal/v25/n4/full/cr201524a.html"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-Cell_Res-ADAR1.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Yin QF#, Hu SB#, Xu YF, <strong>Yang L</strong>, Carmichael GG and Chen LL. SnoVectors for nuclear expression of RNA. <strong>Nucleic Acids Res</strong>, 2015, 43(1): e5 
                <a href="http://nar.oxfordjournals.org/content/43/1/e5"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-Nucleic_Acids_Res-SnoVectors.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Wang D#, Cai C#, Dong X, Yu QC, Zhang XO, <strong>Yang L</strong> and Zeng YA. Identification of multipotent mammary stem cells by protein C receptor expression. <strong>Nature</strong>, 2015, 517(7532): 81-84 
                <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13851.html"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-Nature-MMSC.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Yin QF, Chen LL* and <strong>Yang L*</strong>. Fractionation of Non-polyadenylated and Ribosomal-free RNAs from Mammalian Cells. <strong>Methods Mol Biol</strong>, 2015, 1206: 69-80 (Book chapter) 
                <a href="http://link.springer.com/protocol/10.1007/978-1-4939-1369-5_6"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2015-Methods_Mol_Biol-NonPolyA-Book_chapter.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
        </ul>
        
        <a id="2014" style="position: relative;top: -50px;"></a>
        <h2>2014:</h2>
        <ul class="uk-list-space">
            
            <li><strong>Yang L*</strong> and Chen LL*. Microexons go big. <strong>Cell</strong>, 2014, 159(7): 1488-1489 (Invited preview) 
                <a href="http://www.cell.com/cell/abstract/S0092-8674(14)01574-8"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2014-Cell-microexons_go_big.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li><strong>Yang L*</strong> and Chen LL*. Competition of RNA splicing: line in or circle up. <strong>Sci China Life Sci</strong>, 2014, 57(12): 1232-1233 (Invited review) 
                <a href="http://life.scichina.com:8082/sciCe/EN/10.1007/s11427-014-4770-7"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2014-Sci_China_Life_Sci-line_circle.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Dong R, Chen LL and <strong>Yang L</strong>. Research progress of circular RNA in the post-genome era. <strong>Chinese J Cell Biol</strong>, 2014, 36: 1455-1459 (Invited review, in Chinese) 
                <a href="http://www.cjcb.org/news/upload/20141124-1.pdf"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2014-Chinese_J_Cell_Biol-circular_RNA.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhang XO#, Wang HB#, Zhang Y, Lu X, Chen LL* and <strong>Yang L*</strong>. Complementary sequence-mediated exon circularization. <strong>Cell</strong>, 2014, 159(1): 134-147 (<strong>Issue Highlight</strong>) 
                <a href="http://www.cell.com/cell/abstract/S0092-8674(14)01111-8"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2014_Cell_circRNA.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Editorial by: Vicens Q and Westhof E. <strong>Cell</strong>, 2014, 159: 13-14</li>
                
                <li>Highlighed by: <strong>Nat Rev Genet</strong>, 2014, 15: 707</li>
                
                <li>Featured article</li>
                
            </ul>
             
            <li>Gerstein M#, Rozowsky J#, Yan KK#, Wang D#, Cheng C#, Brown JB#, Davis C#, Hillier L#, Sisu C#, Li JJ#, Pei B#, Harmanci AO#, Duff MO#, Djebali S#, Alexander RP, Alver BH, Auerbach R, Bell K, Bickel PJ, Boech ME, Boley NP, Booth BW, Cherbas L, Cherbas P, Di C, Dobin A, Drenkow J, Ewing B, Fang G, Fastuca M, Feingold EA, Frankish A, Gao G, Good PJ, Guigo R, Hammonds A, Harrow J, Hoskins RA, Howald C, Hu L, Huang H, Hubbard TJP, Huynh C, Jha S, Kasper D, Kato M, Kaufman TC, Kitchen RR, Ladewig E, Lagarde J, Lai E, Leng J, Lu Z, MacCoss M, May G, McWhirter R, Merrihew G, Miller DM, Mortazavi A, Murad R, Oliver B, Olson S, Park PJ, Pazin MJ, Perrimon N, Pervouchine D, Reinke V, Reymond A, Robinson G, Samsonova A, Saunders G, Schlesinger F, Sethi A, Slack FJ, Spencer WC, Stoiber MH, Strasbourger P, Tanzer A, Thompson OA, Wan KH, Wang G, Wang H, Watkins KL, Wen J, Wen K, Xue C, <strong>Yang L</strong>, Yip K, Zaleski C, Zhang Y, Zheng H, Brenner SE*, Graveley BR*, Celniker SE*, Gingeras TR* and Waterston R*. Comparative analysis of the transcriptome across distant species. <strong>Nature</strong>, 2014, 512: 445-448 
                <a href="http://www.nature.com/nature/journal/v512/n7515/full/nature13424.html"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2014-Nature-transcriptome.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhang Y, <strong>Yang L</strong> and Chen LL. Life without A tail: new formats of long noncoding RNAs. <strong>Int J Biochem and Cell Biol</strong>, 2014, 54: 338-349 (Invited review) 
                <a href="http://www.sciencedirect.com/science/article/pii/S1357272513003269"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2014-Int_J_Biochem_and_Cell_Biol-lncRNAs.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhang XO, Yin QF, Chen LL and <strong>Yang L</strong>. Gene expression profiling of non-polyadenylated RNA-seq across species. <strong>Genomics Data</strong>, 2014, 2:237-241 
                <a href="http://www.sciencedirect.com/science/article/pii/S2213596014000634"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2014-Genomics_Data-NonPolyA.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhang XO#, Yin QF#, Wang HB, Zhang Y, Chen T, Zheng P, Lu X, Chen LL* and <strong>Yang L*</strong>. Species-specific alternative splicing leads to unique expression of sno-lncRNAs. <strong>BMC Genomics</strong>, 2014, 15: 287 (<strong>Highly Accessed</strong>) 
                <a href="http://www.biomedcentral.com/1471-2164/15/287/abstract"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2014-BMC_Genomics-sno-lncRNAs.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Editorial by: BioMed Central portal-Biome on 17th April 2014: sno-lncRNAs: a story of splicing across humans, rhesus and mice (http://www.biomedcentral.com/biome/sno-lncrnas-a-story-of-splicing-across- humans-rhesus-and-mice/)</li>
                
            </ul>
             
            <li>Xiang JF, Yin QF, Chen T, Zhang Y, Zhang XO,Wu Z, Zhang S, Wang HB, Ge JH, Lu XH, <strong>Yang L</strong> and Chen LL. Human colorectal cancer-specific CCAT1-L lncRNA regulates long-range chromatin interactions in the MYC locus. <strong>Cell Res</strong>, 2014, 24: 513-531 (<strong>Cover Article and Issue Highlight</strong>) 
                <a href="http://www.nature.com/cr/journal/v24/n5/full/cr201435a.html"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2014-Cell_Res-CCAT1-L.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Editorial by: Younger ST and Rinn JL. <strong>Cell Res</strong>, 2014, 48:155-157</li>
                
                <li>Highlighted by: Nature.com</li>
                
                <li>Highlighted by: Global Medical Discovery</li>
                
                <li>Highlighted by: <strong>National Science Reviews</strong></li>
                
            </ul>
             
        </ul>
        
        <a id="2013" style="position: relative;top: -50px;"></a>
        <h2>2013:</h2>
        <ul class="uk-list-space">
            
            <li>Zhang Y#, Zhang XO#, Chen T, Xiang JF, Yin QF, Xing YH, Zhu S, <strong>Yang L*</strong> and Chen LL*. Circular intronic long noncoding RNAs. <strong>Mol Cell</strong>, 2013, 51: 792-806 (<strong>Issue Highlight</strong>) 
                <a href="http://www.cell.com/molecular-cell/abstract/S1097-2765(13)00590-X"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2013-Mol_Cell-Circular_Intronic.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Editorial by: Bolisetty MT and Graveley BR. <strong>Mol Cell</strong>, 2013, 51:705-706</li>
                
                <li>Editorial by: Reid T. <strong>Nature China</strong>, 2013, Epub. on Oct. 2nd (doi:10.1038)</li>
                
                <li>Editorial by: Tomkins J. Institute for Creation Res, 2013, Epub. on Oct. 9th</li>
                
                <li>Research Highlight by: <strong>Nature</strong>, 2013, 501:464</li>
                
            </ul>
             
            <li>Zhu S#, Xiang JF#, Tian C, Chen LL* and <strong>Yang L*</strong>. Prediction of constitutive A-to-I editing sites from human transcriptomes in the absence of genomic sequences. <strong>BMC Genomics</strong>, 2013,14: 206 (<strong>Highly Accessed</strong>) 
                <a href="http://www.biomedcentral.com/1471-2164/14/206"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2013-BMC_Genomics_Constitutive_A-to-I_editing.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Zhu S, Zhang XO and <strong>Yang L</strong>. Panning for long noncoding RNAs. <strong>Biomolecules</strong>, 2013, 3: 226-241 (Invited review) 
                <a href="http://www.mdpi.com/2218-273X/3/1/226"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2013-Biomolecules-Panning_for_Long_Noncoding_RNAs.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li><strong>Yang L</strong>, Wei G, Tang K, Nardini C and Han JD. Understanding human diseases with high-throughput quantitative measurement and analysis of molecular signatures. <strong>Sci China Life Sci</strong>, 2013, 56: 213-219 (Invited review) 
                <a href="http://life.scichina.com:8082/sciCe/EN/Y2013/V56/I3/213"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2013-SCIENCE_CHINA_Life_Sciences_Review.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
        </ul>
        
        <a id="2012" style="position: relative;top: -50px;"></a>
        <h2>2012:</h2>
        <ul class="uk-list-space">
            
            <li>Bass B, Hundley H, Li JB, Peng Z, Pickrell J, Xiao XG and <strong>Yang L</strong>. The difficult calls in RNA editing. <strong>Nat Biotechnol</strong>, 2012, 30: 1207-1209 (Invited review) 
                <a href="http://www.nature.com/nbt/journal/v30/n12/full/nbt.2452.html"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2012-NBT-RNA_editing.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
             
            <li>Yin QF#, <strong>Yang L#</strong>, Zhang Y, Xiang JF, Wu YW, Carmichael GG* and Chen LL*. Long noncoding RNAs with snoRNA ends. <strong>Mol Cell</strong>, 2012, 48: 219-230 (<strong>Cover Article and Issue Highlight</strong>) 
                <a href="http://www.cell.com/molecular-cell/abstract/S1097-2765(12)00686-7"><i class="uk-icon-external-link"></i></a>
                
                    <a href="/static/publication_pdf/2012-Mol_Cell-snoRNA.pdf"><i class="uk-icon-file-pdf-o"></i></a>
                
            </li>
            
            <ul class="uk-list-space">
                
                <li>Editorial by: McCann KL and Baserga SJ. <strong>Mol Cell</strong>, 2013, 48:155-157</li>
                
                <li>Editorial by: David R. <strong>Nat Rev Mol Cell Biol</strong>, 2012, 13:686</li>
                
                <li>News coverage by: Foundation for Prader-Willi Research</li>
                
                <li>Recommended by: 5 articles in F1000 Prime</li>
                
                <li><strong>Best of Molecular Cell 2012</strong></li>
                
            </ul>
             
        </ul>
        
        <h2 id="before">Before Yang Lab:</h2>
        <div class="uk-panel uk-panel-box">
            <ul>
<li><strong>Yang L</strong>, Duff MO, Graveley BR, Carmichael GG and Chen LL. Genomewide characterization of non-polyadenylated RNAs. <strong>Genome Biol</strong>, 2011, 12: R16  <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-2-r16"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2011_Genome_Biol-NonPolyA.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Graveley BR#, Brooks AN#, Carlson JW#, Duff MO#, Landolin JM#, <strong>Yang L#</strong>, Artieri CG, van Baren MJ, Boley N, Booth BW, Brown JB, Cherbas L, Davis CA, Dobin A, Li R, Lin W, Malone JH, Mattiuzzo NR, Miller D, Sturgill D, Tuch BB, Zaleski C, Zhang D, Blanchette M, Dudoit S, Eads B, Green RE, Hammonds A, Jiang L, Kapranov P, Langton L, Perrimon N, Sandler JE, Wan KH, Willingham A, Zhang Y, Zou Y, Andrews J, Bickel PJ, Brenner SE, Brent MR, Cherbas P, Gingeras TR, Hoskins RA, Kaufman TC, Oliver B and Celniker SE. The developmental transcriptome of Drosophila melanogaster. <strong>Nature</strong>, 2011, 471: 473-479  <a href="https://www.nature.com/articles/nature09715"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2011_Nature-The_developmental_transcriptome_of_Drosophila_melanogaster.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Brooks AN#, <strong>Yang L#</strong>, Duff MO, Hansen KD, Park JW, Dudoit S, Brenner SE and Graveley BR. Conservation of an RNA regulatory map between Drosophila and mammals. <strong>Genome Res</strong>, 2011, 21: 193-202  <a href="https://genome.cshlp.org/content/21/2/193.full"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2011_Genome_Res-Conservation_of_RNA_regulatory_map.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Hoskins RA#, Landolin JM#, Brown JB#, Sandler JE, Takahashi H, Lassmann T, Yu C, Booth BW, Zhang D, Wan KH, <strong>Yang L</strong>, et al. Genome-wide analysis of promoter architecture in Drosophila melanogaster. <strong>Genome Res</strong>, 2011, 21: 182-192  <a href="https://genome.cshlp.org/content/21/2/182.full"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2011_Genome_Res-promoter_architecture_Drosophila_melanogaster.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Cherbas L, Willingham A, Zhang D, <strong>Yang L</strong>, et al. The transcriptional diversity of 25 Drosophila cell lines. <strong>Genome Res</strong>, 2011, 21: 301-314  <a href="https://genome.cshlp.org/content/21/2/301.full"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2011_Genome_Res-25_Drosophila_cell_lines.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Peng SP, Chen LL, Lei XX, <strong>Yang L</strong>, Lin HF, Carmichael GG and HuangYQ. Genome-wide Studies Reveal that Lin28 Enhances the Translation of Genes Important for Growth and Survival of Human Embryonic Stem Cells. <strong>Stem Cells</strong>, 2011, 29: 496-504  <a href="https://stemcellsjournals.onlinelibrary.wiley.com/doi/full/10.1002/stem.591"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2011_Stem_Cells-Lin28.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li><strong>The modENCODE Consortium</strong>. Identification of Functional Elements and Regulatory Circuits by Drosophila modENCODE. <strong>Science</strong>, 2010, 330: 1787-1797  <a href="http://science.sciencemag.org/content/330/6012/1787"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2010_Science-Functional_Elements_Drosophila_modENCODE.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Chen LL*, <strong>Yang L</strong> and Carmichael GG*. Molecular basis for an attenuated cytoplasmic dsRNA response in human embryonic stem cells. <strong>Cell Cycle</strong>, 2010, 9: 3552-3564  <a href="https://www.tandfonline.com/doi/full/10.4161/cc.9.17.12792"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2010_Cell_Cycle-cytoplasmic_dsRNA.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li><strong>The modENCODE Consortium</strong>. Unlocking the secrets of the genome. <strong>Nature</strong>, 2009, 459: 927-930 (Feature article) <a href="https://www.nature.com/articles/459927a"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2010_Nature-Unlocking_the_secrets_of_the_genome.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Chen J, Zhao XN, <strong>Yang L</strong>, Hu GJ, Lu M, Xiong Y, Yang XY, Chang CC, Song BL, Chang TY and Li BL. RNA secondary structures located in the interchromosomal region of human ACAT1 chimeric mRNA are required to produce the 56-kDa isoform. <strong>Cell Res</strong>, 2008, 18: 921-936  <a href="https://www.nature.com/articles/cr200866"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2008_Cell_Res-RNA_secondary_structures_ACAT1.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li><strong>Yang L</strong>, Park J and Graveley BR. Splicing from the Outside In. <strong>Mol Cell</strong>, 2007, 27: 861-862 (Previews) <a href="https://www.sciencedirect.com/science/article/pii/S1097276507005886"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2007_Mol_Cell-Splicing_from_the_Outside_In.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li><strong>Yang L</strong> and Altman S. A non-coding RNA in Saccharomyces cerevisiae is a RNase P substrate. <strong>RNA</strong>, 2007, 13: 682-690  <a href="https://rnajournal.cshlp.org/content/13/5/682.full"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2007_RNA-noncoding_RNA_RNase_P_substrate.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Song BL#, Wang CH#, Yao XM#, <strong>Yang L</strong>, Zhang WJ, Wang ZZ, Zhao XN, Yang JB, Qi W, Yang XY, Inoue K, Lin ZX, Zhang HZ, Kodama T, Chang CC, Liu YK, Chang TY and Li BL. Human acyl-CoA:cholesterol acyltransferase 2 gene expression in intestinal Caco-2 cells and in hepatocellular carcinoma. <strong>Biochem J</strong>, 2006, 394: 617-626  <a href="http://www.biochemj.org/content/394/3/617"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2006_Biochem_J-acyl_CoA_cholesterol_acyltransferase_2.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Kovrigina E, <strong>Yang L</strong>, Pfund E and Altman S. Regulated expression of functional external guide sequences in mammalian cells sing a U6 RNA polymerase III promoter. <strong>RNA</strong>, 2005, 11: 1588-1595  <a href="https://rnajournal.cshlp.org/content/11/10/1588.full"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2005_RNA-U6_RNA_polymerase_III_promoter.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li><strong>Yang L</strong>, Wesolowski D, Li Y and Altman S. Analysis of putative RNase P RNA from orthopoxvirus. <strong>J Mol Biol</strong>, 2005, 354: 529-535  <a href="https://www.sciencedirect.com/science/article/pii/S0022283605010764"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2005_J_Mol_Biol-RNase_P_RNA.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li><strong>Yang L</strong>, Chen J, Chang CC, Yang XY, Wang ZZ, Chang TY and Li BL. A stable upstream stem-loop structure enhances selection of the first 5'-ORF-AUG as a main start codon for translation initiation of human ACAT1 mRNA. <strong>Acta Bioch Bioph Sin</strong>, 2004, 36: 259-268  <a href="https://www.ncbi.nlm.nih.gov/pubmed/15253151/"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2004_Acta_Biochim_Biophys_Sin-stem_loop_structure_first_5'-ORF-AUG.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li><strong>Yang L</strong>, Yang JB, Chen J, Yu GY, Zhou P, Lei L, Wang ZZ, Chang CC, Yang XY, Chang TY and Li BL. Enhancement of human ACAT1 gene expression to promote the macrophage-derived foam cell formation by dexamethasone. <strong>Cell Res</strong>, 2004, 14: 315-323  <a href="https://www.nature.com/articles/7290231"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2004_Cell_Res-ACAT1_macrophage-derived_foam_cell.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li><strong>Yang L</strong>, Lee O, Chen J, Chen J, Chang CC, Wang ZZ, Ma HH, Sha HF, Feng JX, Wang Y, Yang XY, Wang L, Ornvold K, Li BL and Chang TY. Human acyl-coenzyme A:cholesterol acyltransferase 1 (acat1) sequences located in two different chromosomes (7 and 1) are required to produce a novel ACAT1 isoenzyme with additional sequence at the N-terminal. <strong>J Biol Chem</strong>, 2004, 279: 46253-46262  <a href="http://www.jbc.org/content/279/44/46253.full"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2004_J_Biol_Chem-ACAT1_Isoenzyme.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Ma HH, <strong>Yang L</strong> and Li BL. Expression, purification and in vitro N-myristoylation of human Src N-terminal region. <strong>Acta Bioch Bioph Sin</strong>, 2003, 35: 13-17  <a href="http://www.abbs.org.cn/fulltxt/eng/35010013.htm"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2003_Acta_Bioch_Bioph_Sin-Src_N-terminal_region.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Ma HH, <strong>Yang L</strong>, Xu ZP, Yang XY and Li BL. Bacterial expression, purification and in vitro N-myristoylation of fusion hepatitis B virus preS1 with the native type-N-terminal. <strong>Protein Express Purif</strong>, 2003, 27: 49-54  <a href="https://www.sciencedirect.com/science/article/pii/S1046592802005417"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2003_Protein_Express_Purif-fusion_hepatitis_B_virus_preS1.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
<li>Yang JB, Duan ZJ, Yao W, Lee O, <strong>Yang L</strong>, Yang XY, Sun X, Chang CC, Chang TY* and Li BL*. Synergistic transcriptional activation of human ACAT-1 gene by IFN-r and ATRA in THP-1 cells. <strong>J Biol Chem</strong>, 2001, 276: 20989-20999  <a href="http://www.jbc.org/content/early/2001/02/28/jbc.M011488200.full.pdf"><i class="uk-icon-external-link"></i></a> <a href="/static/publication_pdf/Before_Yang_Lab/2001_J_Biol_Chem-ACAT-1_IFN-ATRA.pdf"><i class="uk-icon-file-pdf-o"></i></a></li>
</ul> 
        </div>
    </div>

    <div class="uk-width-medium-1-4 uk-hidden-small">
        <div data-uk-sticky="{top:150}">
            <!--<div class="uk-panel uk-panel-box" data-uk-sticky="{top:150}">-->
            <div class="uk-panel uk-panel-box">
                <h3 class="uk-nav-header">Publication Year</h3>
                <ul class="uk-nav uk-nav-side" data-uk-scrollspy-nav="{closest:'li', smoothscroll:true}">
                     
                    <li><a href="#2023">2023 (8)</a></li>
                     
                    <li><a href="#2022">2022 (8)</a></li>
                     
                    <li><a href="#2021">2021 (10)</a></li>
                     
                    <li><a href="#2020">2020 (7)</a></li>
                     
                    <li><a href="#2019">2019 (11)</a></li>
                     
                    <li><a href="#2018">2018 (8)</a></li>
                     
                    <li><a href="#2017">2017 (10)</a></li>
                     
                    <li><a href="#2016">2016 (7)</a></li>
                     
                    <li><a href="#2015">2015 (11)</a></li>
                     
                    <li><a href="#2014">2014 (9)</a></li>
                     
                    <li><a href="#2013">2013 (4)</a></li>
                     
                    <li><a href="#2012">2012 (2)</a></li>
                    
                    <li class="uk-nav-divider"></li>
                    <li><a href="#before">Before Yang Lab</a></li>
                </ul>
            </div>
            <!--<div class="uk-panel uk-panel-box">-->
                <!--<h3 class="uk-nav-header">Publication type</h3>-->
                <!--<div class="uk-form-select" data-uk-form-select>-->
                    <!--<select>-->
                        <!--<option value="">article</option>-->
                        <!--<option value="">review</option>-->
                    <!--</select>-->
                <!--</div>-->
            <!--</div>-->
        </div>

    </div>

    <div class="uk-width-medium-1-4 uk-hidden-small">
    </div>

</div>


        </article>

        
        <hr class="uk-grid-divider">
<div class="uk-panel uk-text-center">
    <p>Copyright 2012-2023 Bioinformatics and Interdisciplinary Technologies (BITs), Yang Lab. All rights reserved.</p>

</div>
        

    </div>

    <div id="nav-offcanvas" class="uk-offcanvas">
    <div class="uk-offcanvas-bar">
        <ul class="uk-nav uk-nav-offcanvas" data-uk-nav>
            <li><a href="/index"><i class="uk-icon-home"></i> Home</a></li>
            <li><a href="/news"><i class="uk-icon-newspaper-o"></i> News</a></li>
            <li><a href="/awards"><i class="uk-icon-newspaper-o"></i> Awards</a></li>
            <li><a href="/pictures"><i class="uk-icon-picture-o"></i> Pictures</a></li>
            <li><a href="/team"><i class="uk-icon-users"></i> Team</a></li>
            <li><a href="/publications"><i class="uk-icon-file-text"></i> Publications</a></li>
            <li><a href="/resources"><i class="uk-icon-wrench"></i> Resources</a></li>
            <!--<li><a href="/SM"><i class="uk-icon-cubes"></i> Supplementary Materials</a></li>-->
            <li><a href="/login"><i class="uk-icon-user"></i> Login</a></li>
        </ul>
    </div>
</div>

    <script type="text/javascript">
    $('.uk-navbar a:contains("Publications")').parent().addClass('uk-active');
    $('.uk-nav a:contains("Publications")').parent().addClass('uk-active');
    </script>
    <!-- Piwik -->
    <script type="text/javascript">
    var _paq = _paq || [];
    _paq.push(['trackPageView']);
    _paq.push(['enableLinkTracking']);
    (function() {
        var u="//www.picb.ac.cn/rnomics/piwik/";
        _paq.push(['setTrackerUrl', u+'piwik.php']);
        _paq.push(['setSiteId', 1]);
        var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
        g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
    })();
    </script>
    <noscript><p><img src="//www.picb.ac.cn/rnomics/piwik/piwik.php?idsite=1" style="border:0;" alt="" /></p></noscript>
    <!-- End Piwik Code -->

    

</body>

</html>